### PR TITLE
Fix a problem appeared when crossProject imports in other one as a Proje...

### PR DESCRIPTION
**Pull request by yelbota**
...ctRef.

sbt-plugin-test builds correctly.

### CODE REVIEW

**Comment by sjrd** [comment]
Hi,

I'd like to see an [issue](https://github.com/scala-js/scala-js/issues) first, describing what's the *problem*. This PR apparently fixes a problem I am not even aware of.

**Comment by yelbota** [comment]
I encountered with a problem and make this patch. If it necessary I can create an issue and a repo which shows an error with current version of Scala.js.

**Comment by sjrd** [comment]
> I encountered with a problem and make this patch. If it necessary I can create an issue and a repo which shows an error with current version of Scala.js.

Yes, please.

**Comment by yelbota** [comment]
#1569

File: sbt-plugin-test/crossProjectRef/build.sbt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name := "Scala.js sbt test"
+
+version := scalaJSVersion
+
+val versionSettings = Seq(
+  version := scalaJSVersion,
+  scalaVersion := "2.11.6"
```
Comment (sjrd) [inline sbt-plugin-test/crossProjectRef/build.sbt]: Just put these directly in the call to `settings()`

File: sbt-plugin-test/crossProjectRef/project/build.sbt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+addSbtPlugin("org.scala-js" % "sbt-scalajs" %
+  org.scalajs.core.ir.ScalaJSVersions.current)
+
+libraryDependencies += "org.eclipse.jetty" % "jetty-server" % "9.2.3.v20140905"
```
Comment (sjrd) [inline sbt-plugin-test/crossProjectRef/project/build.sbt]: This shouldn't be needed.

File: sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/cross/CrossType.scala [hunk 3]
```diff
@@ -40,25 +40,23 @@ object CrossType {
 
   object Full extends CrossType {
     def projectDir(crossBase: File, projectType: String): File =
-      crossBase.getAbsoluteFile / projectType
+      crossBase / projectType
```
Comment (sjrd) [inline sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/cross/CrossType.scala]: @gzm0 What was the reason for the absolute paths in the first place?

File: sbt-plugin-test/crossProjectRef/build.sbt [hunk 4]
```diff
@@ -0,0 +1,15 @@
+name := "Scala.js sbt test"
+
+version := scalaJSVersion
```
Comment (sjrd) [inline sbt-plugin-test/crossProjectRef/build.sbt]: These two settings are useless, AFAICT.

File: sbt-plugin-test/crossProjectRef/project/build.sbt (hunk 2 above)
Comment (yelbota) [inline sbt-plugin-test/crossProjectRef/project/build.sbt]: libraryDependencies += "org.eclipse.jetty" % "jetty-server" % "9.2.3.v20140905"

Ok

    addSbtPlugin("org.scala-js" % "sbt-scalajs" % org.scalajs.core.ir.ScalaJSVersions.current)

This one is necessary cause it individual project.

**Comment by sjrd** [comment]
The commit message should be something like "Fix #1569: Allow crossProject's JVM and JS parts to be used in a ProjectRef."

Also, the commit should be in a separate branch, i.e., you should not create a PR out of your master branch.

File: sbt-plugin-test/crossProjectRef/project/build.sbt (hunk 2 above)
Comment (sjrd) [inline sbt-plugin-test/crossProjectRef/project/build.sbt]: > This one is necessary cause it individual project.

Yes, sure, my comment was only about the jetty line.

File: sbt-plugin-test/crossProjectRef/build.sbt (hunk 4 above)
Comment (yelbota) [inline sbt-plugin-test/crossProjectRef/build.sbt]: Ok

**Comment by yelbota** [comment]
I have created new PR #1570 follow your recommendations.

**Pull request closed without merging**